src/orion/benchmark/task/quadratics.py

```diff
@@ -0,0 +1,505 @@
+""" Defines the Quadratics task, as described in section 4.2 of the ABLR paper.
+
+[1] [Scalable HyperParameter Transfer Learning](
+    https://papers.nips.cc/paper/7917-scalable-hyperparameter-transfer-learning)
+"""
+from functools import partial
+from typing import ClassVar, Type, Union, List, Dict, Any, Tuple, Dict
+import numpy as np
+import math
+from dataclasses import dataclass, fields, replace
+from simple_parsing.helpers.hparams import HyperParameters, uniform
+from orion.benchmark.task.task import Task, compute_identity
+
+from orion.benchmark.task.utils import similarity, distance
+
+from logging import getLogger as get_logger
+from dataclasses import is_dataclass, asdict
+
+logger = get_logger(__name__)
+
+
+@dataclass
+class QuadraticsTaskHParams(HyperParameters):
+    # TODO: In the paper, section 4.2, the function is defined over R^3, but I'm
```

Comment:
Is there something to do or this is a note?

Reply:
It's a Note. Changed the string from "TODO:" to "NOTE:"